tsktsk/commands/base.py

```diff
@@ -64,13 +70,28 @@ def root(github: Optional[str]) -> None:
 
     github_repository = find_github_repository(config)
     if github_repository:
-        tasks = GithubRepository(github_repository, find_github_auth(config))
+        auth_dao = GithubAuthDao(database())
+        tasks = GithubRepository(github_repository, find_github_auth(config, auth_dao))
+
+    conversation = smalld_click.get_conversation()
+    auth_handler = (
+        GithubAuthHandler(config.github_app_client_id, "public_repo")
+        if conversation
+        else None
+    )
 
-    click.get_current_context().obj = tasks
+    click.get_current_context().obj = {
```

Comment:
with the potential for database to also be linked to this context, we may want to make this some kind of class rather than a dict, although since we wrap all access behind methods it's a bit of a nit.